### PR TITLE
force delimiting on comma

### PR DESCRIPTION
**Pull request by Pomax**
fixes https://github.com/mozilla/network-pulse/issues/562, the ninja hack way.

Ideally, this hack can be taken out again after https://github.com/i-like-robots/react-tags/issues/76 is addressed.

### CODE REVIEW

**Comment by alanmoo** [comment]
Is there a spot we can stick a `trim` so that if a user types `comma space` the new tag doesn't get saved with a leading space?

File: pages/add/form/detail-info-fields.js [hunk 1]
```diff
@@ -22,6 +22,32 @@ let Tags = React.createClass({
...
+      const trueInput = reactTags.input.input;
+      const forceKey = reactTags.handleKeyDown.bind(reactTags);
+
+      trueInput.addEventListener("keyup", evt => {
+        // did we type a comma?
+        const key = evt.key;
+        if (key === ',') {
+          // remove the comma and type an enter instead
+          const { query, selectedIndex } = reactTags.state;
+          reactTags.state.query = query.substring(0, query.length-1).trim();
```
Comment (alanmoo) [inline pages/add/form/detail-info-fields.js]: This isn't the right spot for this, doesn't get triggered on the final tag if the user separates with tab/return

Comment (alanmoo) [inline pages/add/form/detail-info-fields.js]: After line 63 add  `tag.name.trim();`

Comment (mmmavis) [inline pages/add/form/detail-info-fields.js]: I didn't catch ^ and I just R+'ed this PR. Let me take a closer look again.

Comment (mmmavis) [inline pages/add/form/detail-info-fields.js]: like below?

```jsx
updateTags: function(tags) {
    this.setState({ tags }, () => {
      const tagNames = this.state.tags.slice().map(tagObj => tagObj.name.trim());
      this.props.onChange(null,tagNames);
    });
  },
```

File: pages/add/form/detail-info-fields.js [hunk 2]
```diff
@@ -35,7 +64,7 @@ let Tags = React.createClass({
     this.updateTags(tags);
   },
   handleAddition: function(tag) {
-    const tags = [].concat(this.state.tags, tag);
+    const tags = [].concat(this.state.tags, tag.trim());
```
Comment (mmmavis) [inline pages/add/form/detail-info-fields.js]: 👍

Comment (alanmoo) [inline pages/add/form/detail-info-fields.js]: Nope, needs to be `tag.name.trim()` on the line before. `tag` is an object, so we can't directly trim it, and we can't pass `concat` just the name string.

**Comment by mmmavis** [comment]
Got okay from both @alanmoo and @Pomax . I'll merge this on Monday since it's almost EOD Friday 🏖

**Pull request merged**